### ISSUE TITLE
new site init doesn't work

### ISSUE THREAD

**Issue by kritik** [locomotivecms/wagon#250]
log:
```
vladimir-2:rails vladimir$ gem install locomotivecms_wagon --pre --no-ri --no-rdoc
Fetching: faker-1.4.3.gem (100%)
Successfully installed faker-1.4.3
Fetching: yui-compressor-0.12.0.gem (100%)
Successfully installed yui-compressor-0.12.0
Fetching: rack-livereload-0.3.16.gem (100%)
Successfully installed rack-livereload-0.3.16
Fetching: ffi-1.9.10.gem (100%)
Building native extensions.  This could take a while...
Successfully installed ffi-1.9.10
Fetching: rb-inotify-0.9.5.gem (100%)
Successfully installed rb-inotify-0.9.5
Fetching: rb-fsevent-0.9.5.gem (100%)
Successfully installed rb-fsevent-0.9.5
Fetching: hitimes-1.2.2.gem (100%)
Building native extensions.  This could take a while...
Successfully installed hitimes-1.2.2
Fetching: timers-4.0.1.gem (100%)
Successfully installed timers-4.0.1
Fetching: celluloid-0.16.0.gem (100%)
Successfully installed celluloid-0.16.0
Fetching: listen-2.10.1.gem (100%)
Successfully installed listen-2.10.1
Fetching: stringex-2.5.2.gem (100%)
Successfully installed stringex-2.5.2
Fetching: activesupport-4.2.3.gem (100%)
Successfully installed activesupport-4.2.3
Fetching: locomotivecms_common-0.0.4.gem (100%)
Successfully installed locomotivecms_common-0.0.4
Fetching: locomotivecms-liquid-4.0.0.alpha2.gem (100%)
Successfully installed locomotivecms-liquid-4.0.0.alpha2
Fetching: locomotivecms-solid-4.0.0.alpha2.gem (100%)
Successfully installed locomotivecms-solid-4.0.0.alpha2
Fetching: mime-types-2.6.1.gem (100%)
Successfully installed mime-types-2.6.1
Fetching: mimetype-fu-0.1.2.gem (100%)
Successfully installed mimetype-fu-0.1.2
Fetching: haml-4.0.6.gem (100%)

HEADS UP! Haml 4.0 has many improvements, but also has changes that may break
your application:

* Support for Ruby 1.8.6 dropped
* Support for Rails 2 dropped
* Sass filter now always outputs <style> tags
* Data attributes are now hyphenated, not underscored
* html2haml utility moved to the html2haml gem
* Textile and Maruku filters moved to the haml-contrib gem

For more info see:

http://rubydoc.info/github/haml/haml/file/CHANGELOG.md

Successfully installed haml-4.0.6
Fetching: RedCloth-4.2.9.gem (100%)
Building native extensions.  This could take a while...
Successfully installed RedCloth-4.2.9
Fetching: kramdown-1.7.0.gem (100%)
Successfully installed kramdown-1.7.0
Fetching: chunky_png-1.3.4.gem (100%)
Successfully installed chunky_png-1.3.4
Fetching: compass-import-once-1.0.5.gem (100%)
Successfully installed compass-import-once-1.0.5
Fetching: compass-core-1.0.3.gem (100%)
Successfully installed compass-core-1.0.3
Fetching: compass-1.0.3.gem (100%)
    Compass is charityware. If you love it, please donate on our behalf at http://umdf.org/compass Thanks!
Successfully installed compass-1.0.3
Fetching: coffee-script-2.4.1.gem (100%)
Successfully installed coffee-script-2.4.1
Fetching: commonjs-0.2.7.gem (100%)
Successfully installed commonjs-0.2.7
Fetching: less-2.6.0.gem (100%)
Successfully installed less-2.6.0
Fetching: sprockets-less-0.6.1.gem (100%)
Successfully installed sprockets-less-0.6.1
Fetching: sprockets-sass-1.3.1.gem (100%)
Successfully installed sprockets-sass-1.3.1
Fetching: rack_csrf-2.5.0.gem (100%)
Successfully installed rack_csrf-2.5.0
Fetching: moneta-0.8.0.gem (100%)
Successfully installed moneta-0.8.0
Fetching: dragonfly-1.0.10.gem (100%)
================================================================================

You've installed Dragonfly hooray!!

Quite a few things have changed since version 0.9.
Please check the documentation at <http://markevans.github.io/dragonfly>
and see upgrade notes at <https://github.com/markevans/dragonfly/wiki/Upgrading-from-0.9-to-1.0> if upgrading.

================================================================================
Successfully installed dragonfly-1.0.10
Fetching: rack-cache-1.2.gem (100%)
Successfully installed rack-cache-1.2
Fetching: rack-rewrite-1.5.1.gem (100%)
Successfully installed rack-rewrite-1.5.1
Fetching: chronic-0.10.2.gem (100%)
Successfully installed chronic-0.10.2
Fetching: multi_xml-0.5.5.gem (100%)
Successfully installed multi_xml-0.5.5
Fetching: httparty-0.13.5.gem (100%)
When you HTTParty, you must party hard!
Successfully installed httparty-0.13.5
Fetching: morphine-0.1.1.gem (100%)
Successfully installed morphine-0.1.1
Fetching: nokogumbo-1.4.1.gem (100%)
Building native extensions.  This could take a while...
Successfully installed nokogumbo-1.4.1
Fetching: crass-1.0.2.gem (100%)
Successfully installed crass-1.0.2
Fetching: sanitize-4.0.0.gem (100%)
Successfully installed sanitize-4.0.0
Fetching: locomotivecms_steam-1.0.0.pre.alpha.3.gem (100%)
Successfully installed locomotivecms_steam-1.0.0.pre.alpha.3
Fetching: multipart-post-2.0.0.gem (100%)
Successfully installed multipart-post-2.0.0
Fetching: faraday-0.9.1.gem (100%)
Successfully installed faraday-0.9.1
Fetching: faraday_middleware-0.9.2.gem (100%)
Successfully installed faraday_middleware-0.9.2
Fetching: locomotivecms_coal-1.0.0.pre.alpha.3.gem (100%)
Successfully installed locomotivecms_coal-1.0.0.pre.alpha.3
Fetching: netrc-0.10.3.gem (100%)
Successfully installed netrc-0.10.3
Fetching: daemons-1.2.3.gem (100%)
Successfully installed daemons-1.2.3
Fetching: eventmachine-1.0.7.gem (100%)
Building native extensions.  This could take a while...
Successfully installed eventmachine-1.0.7
Fetching: thin-1.6.3.gem (100%)
Building native extensions.  This could take a while...
Successfully installed thin-1.6.3
Fetching: locomotivecms_wagon-2.0.0.pre.alpha.3.gem (100%)
Successfully installed locomotivecms_wagon-2.0.0.pre.alpha.3
51 gems installed
vladimir-2:rails vladimir$ wagon init demo_site
/Users/vladimir/.rvm/gems/ruby-2.2.0/gems/locomotivecms_wagon-2.0.0.pre.alpha.3/lib/locomotive/wagon/generators/site/base.rb:23:in `copy_sources': uninitialized constant Locomotive::Wagon::VERSION (NameError)
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `block in invoke_all'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `each'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `map'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `invoke_all'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/locomotivecms_wagon-2.0.0.pre.alpha.3/lib/locomotive/wagon/commands/init_command.rb:14:in `generate'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/locomotivecms_wagon-2.0.0.pre.alpha.3/lib/locomotive/wagon/commands/init_command.rb:6:in `generate'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/locomotivecms_wagon-2.0.0.pre.alpha.3/lib/locomotive/wagon.rb:29:in `init'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/locomotivecms_wagon-2.0.0.pre.alpha.3/lib/locomotive/wagon/cli.rb:207:in `init'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/locomotivecms_wagon-2.0.0.pre.alpha.3/bin/wagon:12:in `<top (required)>'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/bin/wagon:23:in `load'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/bin/wagon:23:in `<main>'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/bin/ruby_executable_hooks:15:in `eval'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/bin/ruby_executable_hooks:15:in `<main>'
vladimir-2:rails vladimir$ wagon--help
-bash: wagon--help: command not found
vladimir-2:rails vladimir$ wagon --help
Commands:
  wagon auth [EMAIL] [PASSWORD] [PLATFORM_URL]  # Log into your Locomotive platform
  wagon clone NAME HOST [PATH]                  # Clone a remote LocomotiveCMS site
  wagon destroy ENV [PATH]                      # Destroy a remote LocomotiveCMS Engine
  wagon generate RESOURCE ARGUMENTS             # Generates a content_type, page, or snippet
  wagon help [COMMAND]                          # Describe available commands or one specific command
  wagon init NAME [PATH] [GENERATOR_OPTIONS]    # Create a brand new site
  wagon list_templates                          # List all the templates to create either a site or a content type
  wagon pull ENV [PATH]                         # Pull a site from a remote LocomotiveCMS Engine
  wagon push ENV [PATH]                         # Push a site to a remote LocomotiveCMS Engine
  wagon serve [PATH]                            # Serve a site from the file system
  wagon stop [PATH]                             # Stop serving a site previously launched by the serve command with the -d option
  wagon version                                 # Version of the LocomotiveCMS Wagon

Options:
  [--force-color], [--no-force-color]  # Whether or not to use ANSI color in the output.

vladimir-2:rails vladimir$ wagon list_templates
blank - A blank site with the minimal files.
line_case - A simple portfolio site.
bootstrap3 - A site powered by Twitter bootstrap (v3.1.1).
foundation5 - A site powered by Foundation (v5.4.7).
unzip - Unzip a local or remote (http, https, ftp) zipped site.
vladimir-2:rails vladimir$ wagon init demo_site -t blank
/Users/vladimir/.rvm/gems/ruby-2.2.0/gems/locomotivecms_wagon-2.0.0.pre.alpha.3/lib/locomotive/wagon/generators/site/base.rb:23:in `copy_sources': uninitialized constant Locomotive::Wagon::VERSION (NameError)
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `block in invoke_all'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `each'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `map'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:133:in `invoke_all'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/locomotivecms_wagon-2.0.0.pre.alpha.3/lib/locomotive/wagon/commands/init_command.rb:14:in `generate'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/locomotivecms_wagon-2.0.0.pre.alpha.3/lib/locomotive/wagon/commands/init_command.rb:6:in `generate'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/locomotivecms_wagon-2.0.0.pre.alpha.3/lib/locomotive/wagon.rb:29:in `init'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/locomotivecms_wagon-2.0.0.pre.alpha.3/lib/locomotive/wagon/cli.rb:207:in `init'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/thor-0.19.1/lib/thor/command.rb:27:in `run'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/thor-0.19.1/lib/thor/invocation.rb:126:in `invoke_command'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/thor-0.19.1/lib/thor.rb:359:in `dispatch'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/thor-0.19.1/lib/thor/base.rb:440:in `start'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/gems/locomotivecms_wagon-2.0.0.pre.alpha.3/bin/wagon:12:in `<top (required)>'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/bin/wagon:23:in `load'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/bin/wagon:23:in `<main>'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/bin/ruby_executable_hooks:15:in `eval'
	from /Users/vladimir/.rvm/gems/ruby-2.2.0/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by cappadona** [comment]
I was also hitting this `uninitialized constant Locomotive::Wagon::VERSION` error and just submitted #251 that resolves the issue for me.

As I mentioned in the PR, I'm new to Locomotive so please take it with a grain of salt.

**Comment by catalanska** [comment]
+1 I am having this issue too

**Comment by cappadona** [comment]
I just closed #251 since this issue is resolved with b1ff46e. Thanks @did!
